src/main/java/co/cask/plugin/format/MultiTableDBInputFormat.java

```diff
@@ -76,6 +80,15 @@
     }
   }
 
+  private static boolean useTable(String tableName, List<String> whiteList, List<String> blackList) {
+    // If the table name exists in blacklist or when the whiteList is not empty and does not contain table name
+    // the table should not be read
+    if (blackList.contains(tableName) || (!whiteList.isEmpty() && !whiteList.contains(tableName))) {
```

Comment:
can just be `return !blackList.contains(tableName) && (whiteList.isEmpty() || whiteList.contains(tableName))`

Reply:
seeing how its just one line and called just once, can remove the method and just do it inline.

Reply:
Done